docs/advanced-features/event-driven-packaging.md

```diff
@@ -0,0 +1,177 @@
+<!-- markdownlint-disable -->
```

Comment:
I think we should not disable linter.

Reply:
Uff. If we don't I get a multitude of linting errors with one of the tests, especially on text line wraps for list items. Reapplying disable comment.